### PR TITLE
Add utils.js for Code Reduction in index.js file.

### PR DESCRIPTION
**Pull request by RushikeshPatange**
# This PR includes following aspects : 
- **Code reduction** in **index.js** file by introducing **utils.js**  file.
- Moving some unused functions from _index.js_ to _utils.js_ for future scope (if they needed).

@adwait-esp  and @igrr PTAL [here](https://rushikeshpatange.github.io/esp-launchpad/) 😄

### CODE REVIEW

**Review by adwait-esp — APPROVED** [review]

File: js/index.js [hunk 1]
```diff
@@ -194,60 +185,10 @@ $('#device').on('change', function() {
...
-    reader.readAsBinaryString(file);
-}
-
-
-document.getElementById('selectFile1').addEventListener('change', handleFileSelect, false);
-
-function _sleep(ms) {
-    return new Promise(resolve => setTimeout(resolve, ms));
-}
+document.getElementById('selectFile1').addEventListener('change', utilities.handleFileSelect, false);
```
Comment (sachinkosare) [inline js/index.js:191]: The third parameter in addEventListener method anyway defaults to false, right? Just an observation, its okay even if we don't change. But the third parameter specifies if the events capture is required down the DOM tree.

File: js/index.js [hunk 2]
```diff
@@ -397,24 +338,12 @@ addFile.onclick = async () => {
     var btnName = "rem-" + rowCount;
     element3.name = btnName;
     element3.onclick = function() {
-            removeRow(btnName);
+            utilities.removeRow(table,btnName);
```
Comment (sachinkosare) [inline js/index.js:341]: Small observation: use a space after a comma, makes reading code easier.

Comment (RushikeshPatange) [reply js/index.js:341]: Thank you for the feedback! @sachinkosare 😄,Will adopt good practices for code writing from now onwards.

Comment (RushikeshPatange) [reply js/index.js:341]: @sachinkosare The fixes have been merged into this [PR](https://github.com/espressif/esp-launchpad/pull/34).

**Pull request merged**